cafebabel/core/mixins.py

```diff
@@ -45,13 +45,14 @@ def delete_image(self):
     @classmethod
     def store_image(cls, sender, document, **kwargs):
         if document is not None and document._upload_image:
-            document.image_filename = secure_filename(
-                document._upload_image.filename)
+            image_filename = secure_filename(document._upload_image.filename)
+            document.image_filename = (f'/{document.upload_subpath}'
+                                       f'/{image_filename}')
```

Comment:
We're still having the risk of name conflict I guess if 2 pictures have the same name.
I [created an issue](https://github.com/cafebabel/cafebabel.com/issues/257) for this.